.travis.yml

```diff
@@ -155,6 +155,9 @@ before_install:
           if ! php --ri sodium > /dev/null; then
               tfold ext.libsodium tpecl libsodium sodium.so $INI
           fi
+          if ! php --ri mbstring > /dev/null; then
+              tfold ext.mbstring tpecl mbstring mbstring.so $INI
```

Comment:
mbstring is a core extension. Travis should already give us a compiled .so file for it.